circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
I had the same issue locally and had to reset Spotlight for the applications folder. No idea how to fix this on CI.

@AliSoftware or @DanToml: any thoughts?